src/test/auth.spec.js

```diff
@@ -0,0 +1,113 @@
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import app from '../index';
+import TestHelper from '../utils/testHelper';
+import Helper from '../utils/helpers';
+
+chai.use(chaiHttp);
+chai.should();
+
+const URL_PREFIX = '/api/v1/auth';
+
+const apiEndpoint = '/api/v1/auth/login';
+
+const user = {
+  first_name: 'Samuel',
+  last_name: 'koroh',
+  email: 'user1@gmail.com',
+  password: '123456'
+};
+
+const user2 = {
+  first_name: 'Test',
+  last_name: 'Tester',
+  email: 'test@test.com',
+  password: 'PasswordTest123'
+};
+
+describe('/api/v1/auth', () => {
+  let verifiedUser, notVerifiedUser;
+
+  before((done) => {
+    TestHelper.destroyModel('User');
+    done();
+  });
+
+  describe('POST /login', () => {
+    before(async () => {
+      verifiedUser = await TestHelper.createUser({
+        ...user, is_verified: true
+      });
+
+      notVerifiedUser = await TestHelper.createUser({
+        ...user,
+        email: 'user2@gmail.com',
+      });
+    });
+
+    it('should return 400 if the user is not found', async () => {
+      const res = await chai.request(app)
+        .post(apiEndpoint)
+        .set('Content-Type', 'application/json')
+        .send({ email: 'email@email.com', password: 'password' });
+
+      res.should.have.status(400);
```

Comment:
You can also assert that the status message is `error` or `success` depending on what it should return, please apply this in other test cases.